src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -207,6 +207,8 @@ public static bool IsVisible(this MarkdownObject markdownObject)
                     LeafInline _ => true,
                     _ => false,
                 };
+
+                return visible = nodeVisible || visible;
```

Comment:
@yufeih  I think it's caused by this PR: https://github.com/dotnet/docfx/commit/c6ef9ac844b0bd42d2d63bac1b0ed5c543f784b9